contract/r/gnoswap/launchpad/json_builder.gno

```diff
@@ -0,0 +1,118 @@
+package launchpad
```

Comment:
JSON handling will eventually be managed by the chain, allowing most contracts to be significantly simplified by removing all contract-side JSON management (boilerplate).

For this audit, one option is to move JSON into a dedicated package, but this may complicate matters. Therefore, I suggest centralizing all JSON elements in a single file (for each package) and adding a comment for the auditor. This comment should explain that this part is temporary, will be removed, and can ultimately be skipped from the audit or managed as a lower priority.

Reply:
contracts that were not included were excluded either because they had low resuability, or because they encoded specific valuee, not a encoding from specific types.

https://github.com/gnoswap-labs/gnoswap/pull/501
https://github.com/gnoswap-labs/gnoswap/pull/500
https://github.com/gnoswap-labs/gnoswap/pull/495